libs/utils/trace.py

```diff
@@ -70,6 +70,9 @@ def __init__(self, platform, datadir, events,
         # List of events available in the parsed trace
         self.available_events = []
 
+        # Cluster frequency coherency flag
+        self.freq_coherency = True
```

Comment:
It is used in the frequency residency code because we assume all CPUs in a cluster run at the same frequency.